apps/web/src/pages/submission/[step].tsx

```diff
@@ -9,34 +10,70 @@ const Submission = () => {
 
   const step = Number(router.query.step);
 
+  // do not scroll to the top of the form on initial load
+  const formRef = useRef<HTMLDivElement>(null);
+  const prevStepRef = useRef<number>(1);
+  useEffect(() => {
```

Comment:
Would be awesome to have this extracted in to a hook that takes a dependency array and returns a reference to a ref, both for reusability and to reduce component logic